src/db/mongo/client.pl

```diff
@@ -129,6 +131,22 @@
 % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % %
 % % % % % Query Cursor
 
+%% mng_cursor_create(+DB, +Collection, -Cursor) is det.
+%
+% Creates a new query cursor.
+% Make sure to call *mng_cursor_destroy/1* once
+% you are done querying.
+%
+% @param DB The database name
+% @param Collection The collection name
+% @param Cursor The id of a new mongo DB cursor
+%
+mng_cursor_create(DB, Collection, Cursor) :-
+  ( setting(mng_client:tripledb_read_only, true)
+    -> true
```

Comment:
this is not ok.

Cursors must also be created in read-only case, they are used for querying!